scripts/build.js

```diff
@@ -8,6 +8,9 @@ const SimpleIcons = require('simple-icons'),
 // utils
 const { titleToFilename } = require('../lib/utils')
 
+// Exclude the Elsevier icon for now, it seems to be too big
+delete SimpleIcons['Elsevier'];
```

Comment:
We should move that into some configuration with an exclusion property and more importantly introduce proper error handling (tests spotting that the builder lists "invalid" icons), but this should be good for now.

Reply:
> We should move that into some configuration with an exclusion property

Ideally we find a way for all icons to be included into the font 😅 


> We should [...] introduce proper error handling (tests spotting that the builder lists "invalid" icons), but this should be good for now.

Agreed 👍